docs/pages/careers/staff-engineer-pigment-css.md

```diff
@@ -0,0 +1,130 @@
+# Staff Engineer — Pigment CSS
+
+<p class="description">Research, build, document, and help ship a next-gen zero-runtime CSS-in-JS library with a focus on performance and great developer experience.</p>
+
+## Details of the role
+
+- **Location**: Remote (preference for UTC-6 to UTC+5).
+- **Type of work**: Full-time (contractor or employee [depending on circumstances](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501#494af1f358794028beb4b7697b5d3102)).
+- **Level**: [5 or above](https://mui-org.notion.site/Engineering-levels-25b2fba9dd1f4b43a1b28f47f0170f23).
+- We're a **remote** company, we prefer asynchronous communication over meetings.
+
+## The company
+
+MUI's story began in 2014 with Material UI, the most successful React implementation of Google's Material Design.
+Today, Material UI stands as one of the most popular open-source libraries on GitHub and has paved the way for the fully-fledged startup known as MUI (founded in 2019), which now boasts an ever-expanding ecosystem of React UI products.
+We're a company of 31+ people as of late 2023, and we're growing.
+
+## The products
+
+MUI is best known for our flagship product, Material UI—but this is just one of three core component libraries we maintain.
+Base UI is our headless component library, and Pigment CSS is a zero-runtime CSS-in-JS library, that is a replacement of the current styling engine that uses emotion/styled-components under the hood.
+We also host Design Kits and pre-built Templates.
```

Comment:
```suggestion
MUI is best known for our flagship product, Material UI—but this is just one of three core component libraries we maintain.
Base UI is our headless component library, and Pigment CSS is a zero-runtime CSS-in-JS styling engine.
We also host Design Kits and pre-built Templates.
```

I don't think we should sell Pigment early here as a "replacement to Material's (which is implied in the original) current styling engine". Also, this is part of the job ad template copy, and whenever we're past the v6/v7 cycle, this "current" part should become outdated.